CHANGELOG.md

```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.3.0] 2020-03-17
+
+## Changed
+
+- Remove error from the spec.
```

Comment:
We talked about this in last SIG operator. This part should land in a separate PR titled "release v0.3.0". This is for the future. Thanks for sorting that out.